NuRadioReco/modules/io/event_parser_factory.py

```diff
@@ -93,6 +93,10 @@ def scan_files_2_2(self, iF, current_byte):
                     'stations': {}
                 }
             if is_generic_detector:
+                if 'default_station' not in detector_dict:
```

Comment:
Why do you need to explicitly set this to none?

Reply:
@christophwelling maybe we can just remove the explicit setting and also ll 100/101 where they are put into the detector dicts? what do you think?

Reply:
These entries need to exist for NuRadioRecoio to recognise that it is a genericDetector. As long as we want to support older files, this seems to be a decent solution.

Reply:
ok added a comment.